README.md

```diff
@@ -414,6 +422,14 @@ df.write
   .save()
 ```
 
+### Configuring column encoding
+
+When creating a table, `spark-redshift` can be configured to use a specific compression encoding on individual columns. You can use the `encoding` column metadata field to specify a compression encoding for each column (see [Amazon docs](http://docs.aws.amazon.com/redshift/latest/dg/c_Compression_encodings.html) for available encodings).
```

Comment:
This library is undergoing a slight naming / branding change in the README and documentation, so `spark-redshift` needs to be replaced with "this library" (see #179).